Code/Framework/AzFramework/Platform/Common/Xcb/azframework_xcb_files.cmake

```diff
@@ -6,6 +6,8 @@
 #
 #
 
+
+
```

Comment:
Since you refactored out the SPLASH_FILE detection, you can probably revert all changes to this file completely.